core/utils.py

```diff
@@ -22,3 +24,25 @@ def wrap(*args, **kwargs):
             message += ' '.join(('%s=%s' % (str(k), str(v)) for k, v in kwargs.items()))
             return fn(message)
         return wrap
+
+
+class BootstrapChecker:
+    """
+    This is a small helper class to find any problems with missing values
+    in Django configuration.
+    """
+    def __init__(self) -> None:
+        pass
+
+    def _check_idservice(self):
+        idservice_endpoint = getattr(settings, 'IDSERVICE_ENDPOINT', None)
+        # TODO: check ID service endpoint connectivity
+        # i.e.: make http GET request 
+
+        # Checks if the function can be importable
```

Comment:
small typo - can be fixed once the method is finished

Reply:
Ahh, adding this file was a mistake, it is by no means ready